quickstarts/kubecost/config.yml

```diff
@@ -0,0 +1,60 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: kubecost
+
+# Displayed in the UI (required)
+title: Kubecost
+
+# Long-form description of the quickstart (required)
+description: |
+  ## Quickstart for Kubecost Cost Monitoring
+  Kubecost provides real-time cost visibility and insights for teams using Kubernetes, helping you continuously reduce your cloud costs.
+  The link to Github repository provides instructions on how to send kubernetes cost metrics and build visualizations inside New Relic.
```

Comment:
Reworded to - 
The link to Github repository provides instructions on how to send kubernetes cost metrics and build visualizations inside New Relic.